app/views/open_studios_subdomain/artists/show.html.slim

```diff
@@ -4,14 +4,14 @@
       | Artist:
       span.artist__name< #{@artist.get_name}
       - if @artist.doing_open_studios?
-        a href="/open_studios" title="I'm doing Open Studios"
+        = link_to open_studios_url(subdomain: ""), title: "I'm doing Open Studios" do
           .os-violator
 .pure-g.alpha.omega
   .pure-u-1-1.pure-u-sm-2-3.pure-u-md-1-2.pure-u-lg-3-5#art
     .pure-g
       .pure-u-1-1
       - if @artist.art_pieces.present?
-        = render partial: "/open_studios/art", collection: @artist.art_pieces, as: :art_piece, locals: { class_overrides: "pure-u-1-1 pure-u-lg-1-2 pure-u-xl-1-3"}
+        = render partial: "/open_studios_subdomain/artists/art", collection: @artist.art_pieces, as: :art_piece, locals: { class_overrides: "pure-u-1-1 pure-u-lg-1-2 pure-u-xl-1-3"}
```

Comment:
could/should this be a relative path?  "./artists/art"?  or maybe that's too confusing - this is if nothing else explicit.

Reply:
I'm used to seeing relative paths. I was just following the existing pattern. We can change this.

Reply:
Never mind... That broke the feature test. I think there may be some configging we have to do with Capybara 🤔